### ISSUE TITLE
Incorrect treatment of disabled value when creating/updating a user

### ISSUE THREAD

**Issue by udAL** [kreait/firebase-php#196]
#### 1. Which version of `kreait/firebase-php` are you using?
4.6.0

#### 2. Which version of PHP are you using?
7.2.4

#### 3. What's the issue?
When creating or updating a user (createUser || updateUser) if I add the 'disabled' parameter (whatever value) it disables the user. Once disabled there's no way to activate it (tried no parameter, parameter to false, to 0..)

#### 4. Code that lead to the issue
```php
require_once('vendor/autoload.php');

use Kreait\Firebase\Factory;
use Kreait\Firebase\ServiceAccount;

$ServiceAccount = ServiceAccount::fromJson(
    '{
        "type": "---",
        "project_id": "---",
        "private_key_id": "---",
        "private_key": "---",
        "client_email": "---",
        "client_id": "---",
        "auth_uri": "---",
        "token_uri": "---",
        "auth_provider_x509_cert_url": "---",
        "client_x509_cert_url": "---"
    }'
);
$Firebase = (new Factory)
    ->withServiceAccount($ServiceAccount)
    ->create();
$Auth = $Firebase->getAuth();

$userProperties = [
    'displayName' => 'Name',
    'disabled' => true,
    'customAttributes' => array(
        '1' => 1.1,
        '2' => 2.1
    )
];
$User = $Auth->createUser($userProperties);

$userProperties = [
    'displayName' => 'Name2',
    'disabled' => false,
    'customAttributes' => array(
        '1' => 1.2,
        '2' => 2.2
    )
];
$Auth->updateUser($User->uid, $userProperties);

// Result = Disabled user
```

**Comment by jeromegamez** [comment]
Hey @udAL, and thank you for finding and reporting this issue!

I just released a new release ([4.7.1](https://github.com/kreait/firebase-php/releases)) that includes a fix and a test to ensure that it stays that way (https://github.com/kreait/firebase-php/commit/e7d073057c4318825392d696412058dcae8c6146#diff-b16b10233efe1e9e75b6b378959416b7).

**Comment by udAL** [comment]
Man, that was fast...
Thanks!
